ban/db/fields.py

```diff
@@ -199,6 +201,21 @@ def coerce(self, value):
         return value
 
 
+class IPAddressField(CharField):
+
+    max_length = 100
```

Comment:
I wonder if we should keep some generic 100 or try to make it 39 or 45 (http://stackoverflow.com/a/1076755/328117)